scripts/init-env.sh

```diff
@@ -6,7 +6,7 @@ PREFIX=$1
 rm $ENV_FILE 2> /dev/null
 touch $ENV_FILE
 
-for VARIABLE in API_KEY APP_ID AUTH_DOMAIN MEASUREMENT_ID BUCKET_URL PROJECT_ID STRIPE_PUBLISHABLE_KEY BUGSNAG_API_KEY LOGROCKET_APP_ID MIXPANEL_PROJECT_TOKEN WS_RELAY_URL CODE_CHECK_URL IS_BURN
+for VARIABLE in API_KEY APP_ID AUTH_DOMAIN MEASUREMENT_ID BUCKET_URL PROJECT_ID STRIPE_PUBLISHABLE_KEY BUGSNAG_API_KEY LOGROCKET_APP_ID MIXPANEL_PROJECT_TOKEN WS_RELAY_URL CODE_CHECK_URL IS_BURN AGORA_APP_ID AGORA_CHANNEL AGORA_TOKEN
```

Comment:
We won't need any of these here when using the real token fetching from backend method I wrote, so can just revert this back to how it was.

```suggestion
for VARIABLE in API_KEY APP_ID AUTH_DOMAIN MEASUREMENT_ID BUCKET_URL PROJECT_ID STRIPE_PUBLISHABLE_KEY BUGSNAG_API_KEY LOGROCKET_APP_ID MIXPANEL_PROJECT_TOKEN WS_RELAY_URL CODE_CHECK_URL IS_BURN
```